### PR TITLE
Editorial: use the Infra Standard for URL's path

### LINKED ISSUE #235: Use Infra more
To define URL's path, to use return rather than terminate these steps, ...

### PR DESCRIPTION
**Pull request by annevk**
Also stop saying "terminate these steps" and simply return.

Fixes #235.

### CODE REVIEW

File: url.bs [hunk 1]
```diff
@@ -864,9 +864,9 @@ It is initially the empty string.
 <p>A  <a for=/>URL</a>'s <dfn export for=url id=concept-url-port>port</dfn> is either
 null or a 16-bit unsigned integer that identifies a networking port. It is initially null.
 
-<p>A <a for=/>URL</a>'s <dfn export for=url id=concept-url-path>path</dfn> is a list of zero or more
-<a>ASCII strings</a> holding data, usually identifying a location in hierarchical form. It is
-initially the empty list.
+<p>A <a for=/>URL</a>'s <dfn export for=url id=concept-url-path>path</dfn> is a <a>list</a> of zero
```
Comment (domenic) [inline url.bs]: This isn't linking for some reason: https://url.spec.whatwg.org/branch-snapshots/annevk/infra-wip/#concept-url-path

Comment (annevk) [inline url.bs]: I guess I need for=/ since I already have a `<dfn>` for another list.

Comment (annevk) [inline url.bs]: Thanks for noticing!

**Comment by annevk** [comment]
@domenic did you review the whole changeset? Would be nice to land this to avoid bitrot.

**Comment by domenic** [comment]
Yeah, LGTM

**Pull request merged**